Tasks/JenkinsQueueJobV2/jenkinsqueuejobtask.ts

```diff
@@ -2,10 +2,7 @@
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
 import tl = require('vsts-task-lib/task');
-import fs = require('fs');
 import path = require('path');
-import shell = require('shelljs');
```

Comment:
These were unused imports.

Reply:
Can you change the imports here to use `import-from`?